<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1098</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Travel DNPHL - Quản lý Tour Du Lịch</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #ffffff;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="sidebar" native="true">
      <property name="minimumSize">
       <size>
        <width>250</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>250</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #126E34;</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>15</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>15</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>15</number>
       </property>
       <item>
        <widget class="QWidget" name="logo_container" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>60</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #ffffff;</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="logo_text">
            <property name="font">
             <font>
              <family>Arial</family>
              <pointsize>14</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: #ffffff;</string>
            </property>
            <property name="text">
             <string>TRAVEL ĐNPHL</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lbl_chuc_nang">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">color: #ffffff;
padding-left: 15px;</string>
         </property>
         <property name="text">
          <string>CHƯƠNG TRÌNH QUẢN LÝ</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_staff">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="layoutDirection">
          <enum>Qt::LeftToRight</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    color: white;
    background-color: transparent;
    border: none;
    text-align: left;
    padding-left: 15px;
}
QPushButton:hover {
    background-color: #1B8A47;
}
QPushButton:pressed {
    background-color: #1D9F4E;
}</string>
         </property>
         <property name="text">
          <string>Nhân viên</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../assets/user.svg</normaloff>../assets/user.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_customers">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    color: white;
    background-color: transparent;
    border: none;
    text-align: left;
    padding-left: 15px;
}
QPushButton:hover {
    background-color: #1B8A47;
}
QPushButton:pressed {
    background-color: #1D9F4E;
}</string>
         </property>
         <property name="text">
          <string>Khách hàng</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../assets/users.svg</normaloff>../assets/users.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_tours">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    color: white;
    background-color: transparent;
    border: none;
    text-align: left;
    padding-left: 15px;
}
QPushButton:hover {
    background-color: #1B8A47;
}
QPushButton:pressed {
    background-color: #1D9F4E;
}</string>
         </property>
         <property name="text">
          <string>Tour</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../assets/map.svg</normaloff>../assets/map.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_bookings">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    color: white;
    background-color: transparent;
    border: none;
    text-align: left;
    padding-left: 15px;
}
QPushButton:hover {
    background-color: #1B8A47;
}
QPushButton:pressed {
    background-color: #1D9F4E;
}</string>
         </property>
         <property name="text">
          <string>Danh sách đặt tour</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../assets/calendar.svg</normaloff>../assets/calendar.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_statistics">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>40</height>
          </size>
         </property>
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    color: white;
    background-color: transparent;
    border: none;
    text-align: left;
    padding-left: 15px;
}
QPushButton:hover {
    background-color: #1B8A47;
}
QPushButton:pressed {
    background-color: #1D9F4E;
}</string>
         </property>
         <property name="text">
          <string>Thống kê doanh thu</string>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>../assets/bar-chart-2.svg</normaloff>../assets/bar-chart-2.svg</iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
    color: white;
    background-color: transparent;
    border: none;
    text-align: left;
	padding-left: 1px;
}
</string>
         </property>
         <property name="text">
          <string>Lập trình ứng dụng với Python</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
    color: white;
    background-color: transparent;
    border: none;
    text-align: left;
	padding-left: 1px;
}
</string>
         </property>
         <property name="text">
          <string>Nhóm 7 - DCCNTT13.10.6</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3">
         <property name="font">
          <font>
           <family>Arial</family>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">QLabel {
    color: white;
    background-color: transparent;
    border: none;
    text-align: left;
	padding-left: 1px;
}
</string>
         </property>
         <property name="text">
          <string>Giảng viên: Nguyễn Đức Thiện (B)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="content_area" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QStackedWidget" name="stackedWidget"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
